Modules/_asynciomodule.c

```diff
@@ -193,6 +193,49 @@ is_coroutine(PyObject *coro)
     return has_it;
 }
 
+/*[clinic input]
+_asyncio.isfuture
+
+    obj: object
```

Comment:
Make the argument positional-only.

Reply:
In pratice, it means adding "/" on a new line, aligned with "obj", and run "make clinic" again.